src/components/DataDimension/DndContext.js

```diff
@@ -0,0 +1,165 @@
+import {
+    DndContext,
+    DragOverlay,
+    useSensor,
+    useSensors,
+    PointerSensor,
+} from '@dnd-kit/core'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import DraggingItem from './DraggingItem.js'
+
+const activateAt15pixels = {
```

Comment:
weird name. also, why does it need to be a const? (only used once and it's not that complex)